sagemaker-python-sdk/tensorflow_script_mode_quickstart/tensorflow_script_mode_quickstart.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using TensorFlow in SageMaker - Quickstart\n",
+    "\n",
+    "Starting by the TensorFlow's framework version 1.11, you can use the SageMaker TensorFlow Container to train any TensorFlow script. \n",
```

Comment:
we already call it "Script Mode" in the SDK documentation - I think we might as well say it explicitly here rather than leaving readers wondering what TF scripts couldn't be run before